nemo/collections/tts/torch/data.py

```diff
@@ -333,7 +336,7 @@ def get_log_mel(self, audio):
 
     def __getitem__(self, index):
         sample = self.data[index]
-        audio_stem = Path(sample["audio_filepath"]).stem
+        audio_path_as_text_id = sample["audio_filepath"].replace("/", "-").split(".")[0]
```

Comment:
Maybe do audio_stem + deterministic has of the whole path?
Not sure which is better but storing the whole path is pretty extreme.

Reply:
Agree, I thought about it for a long time. Will try to suggest something else.